### PR TITLE
Added Highlight media details

### PR DESCRIPTION
**Pull request by rushmedev**
Added feature to get all Highlight Reels by providing highlight ID `ex: highlight:18122139151093883`.

### CODE REVIEW

File: src/main/java/com/github/instagram4j/instagram4j/responses/highlights/HighlightsMediaResponse.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.github.instagram4j.instagram4j.responses.highlights;
```
Comment (jvogit) [inline src/main/java/com/github/instagram4j/instagram4j/responses/highlights/HighlightsMediaResponse.java:1]: Is it possible to use this response for `FeedReelsMediaRequest` as well?

If so, all is needed is to change the response for `FeedReelsMediaRequest` to use this response. Also following convention the response should be named `FeedReelsMediaResponse`.

The HighlightsMediaRequest is similar to FeedReelsMediaRequest so it's not needed to add this redundant request and should be removed.

**Review by jvogit — CHANGES_REQUESTED** [review]
Thanks for submitting a PR. I appreciate it! Some changes are needed.

File: src/main/java/com/github/instagram4j/instagram4j/responses/highlights/HighlightsMediaResponse.java (hunk 1 above)
Comment (rushmedev) [reply src/main/java/com/github/instagram4j/instagram4j/responses/highlights/HighlightsMediaResponse.java]: Modified the files as per standard of Instagram4j. Now response object is `FeedReelsMediaResponse` .

**Review by jvogit — APPROVED** [review]
Appreciate those changes, great work!

**Pull request merged**